### DISCUSSION TITLE
yiwen-blog-website/docs/backend/java/springboot3_jwt

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#110] [Announcements]
# yiwen-blog-website/docs/backend/java/springboot3_jwt

参考文档

https://hao-yiwen.github.io/yiwen-blog-website/docs/backend/java/springboot3_jwt

**Comment by Hao-yiwen** [comment]
目前基于用户名来注册 这块需要优化
